contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -54,22 +54,25 @@
                 @focus="trackClick('Description')"
               />
```

Comment:
With the computed `mdValue` on the `VFlex` below, perhaps move the `v-if="oneSelected"` on the description field to it's parent `VFlex`